myna/database/peregrine.py

```diff
@@ -49,11 +49,9 @@ def load(self, metadata_type, part=None, layer=None):
         if metadata_type == metadata.LaserPower:
             datafile = os.path.join(self.path, "simulation", part, "part.npz")
             with np.load(datafile, allow_pickle=True) as data:
-                index = [
-                    ind
-                    for ind, x in enumerate(data["parameter_names"])
-                    if x == "Power (W)"
-                ][0]
+                synonyms = ["Laser Beam Power (W)", "Power (W)"]
+                matches = [syn in data["parameter_names"] for syn in synonyms]
```

Comment:
Could use the same util function?

Reply:
Looks good to me! Cleaner and more consistent than what was there.